src/main/fc/fc_init.c

```diff
@@ -236,6 +236,13 @@ void spiPreInit(void)
 #if defined(USE_RX_SPI) && !defined(USE_RX_SOFTSPI)
     spiPreInitCs(IO_TAG(RX_NSS_PIN));
 #endif
+#if defined(LPS_CS_PIN)
+    ////spiPreInitCs(IO_TAG(LPS_CS_PIN));
+    IO_t lps = IOGetByTag(IO_TAG(LPS_CS_PIN));
```

Comment:
Why isn't this using `spiPreInitCs()`?

Reply:
On some of the boards where I had this LPS25HB with preInitCs it wasn't working. I think the configuration set by this routine was different then when I was testing. I'll change it. 

I see that Travis is complaining, I need to remove all the warnings, correct?

Reply:
Yes.